flink-cdc-connect/flink-cdc-source-connectors/flink-connector-mongodb-cdc/pom.xml

```diff
@@ -69,7 +69,7 @@ limitations under the License.
         <dependency>
             <groupId>org.mongodb</groupId>
             <artifactId>mongodb-driver-sync</artifactId>
-            <version>4.9.1</version>
+            <version>5.1.1</version>
```

Comment:
Could you also update the docs ？ and the older(release-3.1) version docs seem use an incorrect driver version.
https://nightlies.apache.org/flink/flink-cdc-docs-release-3.1/docs/connectors/flink-sources/overview/